utils/triage_team.py

```diff
@@ -0,0 +1,55 @@
+# One time script for creating the triage team, and adding the necessary repos to the team
+
+import os
+import asyncio
+import aiohttp
+
+from gidgethub.aiohttp import GitHubAPI
+
+import cachetools
+
+cache = cachetools.LRUCache(maxsize=500)
+
+
+async def get_core_repos(gh, team_id):
+    """
+    Return the team's public repos
+    """
+    async for repo in gh.getiter(f"/teams/{team_id}/repos"):
+        if not repo["private"]:
```

Comment:
Does GitHub have any other keys beyond "private" and "public" for defining repositories? If so (or if they decide to add more distinctions in the future for any reason), it might be a slight improvement to explicitly check if the repo is public rather than not private:

```suggestion
        if repo["public"]:
```